### PR TITLE
Update paths to Quant models

### PR DESCRIPTION
**Pull request by wozna**
### PR types

Bug fixes
### PR changes

Docs 
### Describe

This PR:
- updates paths to Quant models 
- adds paths to models that otherwise obtain scales for quantization such as `ResNet50_qat_range` or `ResNet50_qat_channelwise`

### CODE REVIEW

File: python/paddle/fluid/contrib/slim/tests/README.md [hunk 1]
```diff
@@ -207,13 +207,30 @@ Run the following commands to download and extract Quant model:
...
+export QUANT_MODEL_ARCHIVE=${QUANT_MODEL_NAME}.tar.gz
+wget http://paddle-inference-dist.bj.bcebos.com/int8/QAT_models/${QUANT_MODEL_ARCHIVE}
+mkdir ${QUANT_MODEL_NAME} && tar -xvf ${QUANT_MODEL_ARCHIVE} -C ${QUANT_MODEL_NAME}
+```
+
+To download other Quant models, set the `QUANT_MODEL_NAME` variable to on of the values:
+- `ResNet50_qat_perf`, with input/output scales in `fake_quantize_moving_average_abs_max` operators, with weight scales in `fake_dequantize_max_abs` operators
+- `ResNet50_qat_range`, with input/output scales in `fake_quantize_range_abs_max` operators and the `out_threshold` attributes, with weight scales in `fake_dequantize_max_abs` operators
+- `ResNet50_qat_channelwise`, with input/output scales in `fake_quantize_range_abs_max` operators and the `out_threshold` attributes, with weight scales in `fake_channel_wise_dequantize_max_abs` operators
+- `MobileNet_qat_perf`.
```
Comment (lidanqing-vv) [inline python/paddle/fluid/contrib/slim/tests/README.md]: This line means MobileNet_qat_perf has same rule as ResNet50_qat_perf to change names and then download MobileNet related models successfully?

Comment (lidanqing-vv) [reply python/paddle/fluid/contrib/slim/tests/README.md]: I mean line 233 looks like single line you may delete it. Thanks

**Comment by lidanqing-vv** [comment]
LGTM.

**Comment by wozna** [comment]
@luotao1  Could you please start your review?

**Review by luotao1 — APPROVED** [review]

**Pull request merged**